bin/install-platform

```diff
@@ -88,9 +88,9 @@ helm ${ACTION} \
   --set tags.kubed=${kubed} \
   --set global.postgresqlEnabled=${postgresql} \
   --set tags.keda=${keda} \
+  --set astronomer.houston.upgradeDeployments.enabled=true \
```

Comment:
This line enables the upgrader to run in CI. The upgrader also has the post-install hook.